### ISSUE TITLE
[Task]: Model training

### ISSUE THREAD

**Issue by Annytomica** [Annytomica/cherry-mildew-detector#72]
Epic: #8 
User Story: #67 

### Description:
Train model on augmented training data

### Checklist:
- [x] Train model using tuned hyperparameters and image-size
- [x] save and export CNN architecture diagram
- [x] save and export model
- [x] evaluate performance with loss and accuracy plots
- [x] save and export loss and accuracy plots
